src/main.rs

```diff
@@ -87,10 +105,23 @@ fn main() {
         warn!("could not enable telemetry: {}", err);
     };
 
+    // Handle `oasis chain` before args are parsed so that we can get the raw
+    // args to pass to `oasis-chain`. Among other things, Clap won't allow
+    // collecting unknown flags.
```

Comment:
It doesn't support `--`?  That's been part of IEEE Std 1003.1 forever.

Reply:
it does, but you can't have unknown args _without_ `--` https://github.com/clap-rs/clap/issues/873#issuecomment-436546860